client/router.coffee

```diff
@@ -48,7 +44,7 @@ Router.route "/admins",
   onBeforeAction: ->
     redirectIfNotAuthorized(@, ['admin'])
   waitOn: ->
-    Meteor.subscribe "allUsers"
+    [Meteor.subscribe("allUsers"), Meteor.subscribe("roles")]
```

Comment:
Just a comment: I wonder if we should consider moving this to the controller in the future?